backend/routes/stocks.js

```diff
@@ -1,29 +1,38 @@
 import express from "express";
 import { Stock } from "../models/stock.js";
+import { Follow } from "../models/follow.js";
 
 const router = express.Router();
 
 // Create a new stock in database
 router.post("/stocks", async (req, res) => {
     try {
         const stock = await Stock.create(req.body);
-        res.json(stock);
+        res.json({ stock });
     } catch (error) {
-        res.send(error);
+        res.send({ error });
     }
 });
 
 // Get a specific stock from database
 router.get("/stocks/:ticker", async (req, res) => {
     const ticker = req.params.ticker.toUpperCase();
-    try {
```

Comment:
Why is the try-catch no longer necessary?

Reply:
Stock and follow routes should be fixed on upcoming pull request.